_data/schedule.yml

```diff
@@ -0,0 +1,26 @@
+events:
+  -
+    title: Roundtable
+    location: CreatorSpace (typically)
+    address: 320 Railroad Ave.
+    topic:The first part of the meeting is educational with a speaker invited to
+      present on a niche topic that is of interest to entrepreneurs.
+
+      The second half of the meeting is an opportunity for an entrepreneur to
+      gain collective input from the group.  The entrepreneur has the
+      opportunity to present a new business idea, problem with their business,
+      or area where they are seeking help to a group of fellow entrepreneurs. The
+      group then has the chance to provide feedback and ask questions of
+      the presenter.
```

Comment:
The repeated use of the word "opportunity" might bear rephrasing, and avoiding grammatically controversial pronoun usage might be advisable as well.  Typically, rewriting to remove singular uses of plural pronouns also offers a chance to eliminate some passive-voice phrasing, as in this case.  Perhaps that paragraph could be rewritten in a manner similar to this:

The second half of the meeting provides a forum for an entrepreneur to solicit
collective input from the group.  The entrepreneur has the opportunity to
present a new business idea, business model or resourcing challenge, or other
area where help is needed, to a group of fellow entrepreneurs.  The group then
has the chance to provide constructive feedback and ask questions of the 
presenter that help refocus the entrepreneur's thinking.